docs/src/pages/docs/installation/kubernetes.mdx

```diff
@@ -94,25 +94,20 @@ postgresql:
 
 #### Dependencies
 
-You can specify pip packages to be installed before startup, e.g. to install extra database drivers:
+Install additional packages and do any other bootstrap configuration in this script For production clusters it's
+recommended to build own image with this step done in CI The following example installs the Big Query and Elasticsearch
```

Comment:
and another one here: `... step done in CI. The following ...`

Reply:
@shicholas if you can add this last missing period I can hit merge.